### PR TITLE
Package metadata [WIP - do not merge]

### PR DESCRIPTION
**Pull request by mrshll1001**

### CODE REVIEW

File: documentation/reference.md [hunk 1]
```diff
@@ -287,19 +303,25 @@ The 360Giving JSON Schemas are the authoritative source of information about the
...
+<div style="height:400px; overflow:auto; border:1px solid grey;">
+<script src="../_static/docson/widget.js"
+        data-schema="../360-giving-package-schema.json">
 </script>
 </div>
 
 ### Field names and titles
 
-Each entity, property and relationship in the schema has both a machine-readable field name and an English language title (apart from Transactions).
+Each entity, property and relationship in each the 360Giving Grant shcema and 360Giving Package schema has both a machine-readable field name and an English language title (apart from Transactions in the Grant schema).
```
Comment (robredpath) [inline documentation/reference.md:324]: typo - shcema

File: schema/360-giving-package-schema.json [hunk 2]
```diff
@@ -5,10 +5,100 @@
...
+          "title": "Publisher",
+          "description": "Information to uniquely identify the publisher of this package.",
+          "type": "object",
+          "required": [
+            "name"
+          ],
+          "properties": {
+            "identifier": {
+              "title": "identifier",
+              "description": "The organisation identifier as descriibed by [org-id.guide](http://org-id.guide/)",
```
Comment (robredpath) [inline schema/360-giving-package-schema.json:63]: typo - descriibed

File: schema/360-giving-package-schema.json [hunk 3]
```diff
@@ -5,10 +5,100 @@
...
+              "type": "string"
+            },
+            "name": {
+              "title": "Name",
+              "description": "The name of the organisation or department responsible for publishing this data.",
```
Comment (robredpath) [inline schema/360-giving-package-schema.json:68]: Check whether we use the s- or z- form of organisation

File: documentation/reference.md [hunk 4]
```diff
@@ -39,6 +40,21 @@ The 360Giving Spreadsheet template consists of a 'grants' sheet which contains t
 
 The [Additional fields](additional-fields) section provides details of all other possible fields that can be reported. (These are derived from the [360Giving JSON Schema](360giving-json-schemas) ).
 
+### Meta Sheet
+
+The 'Meta' sheet includes space to add metadata about the grants file. You can read more about metadata in the [Data and Metadata section](#data-and-metadata). The 'meta' sheet includes sections for:
```
Comment (robredpath) [inline documentation/reference.md:45]: 'meta' and 'Meta' appear here. Is this case-sensitive?

**Comment by robredpath** [comment]
[Normative Metadata Documentation](https://trello.com/c/5rtlw0UX/8-normative-metadata-documentation)

**Comment by robredpath** [comment]
[Normative Metadata Documentation](https://trello.com/c/5rtlw0UX/8-normative-metadata-documentation)

**Comment by robredpath** [comment]
Reviewed with @mrshll1001 today. This is getting close to being ready. 

The change has been approved by the SC, so once this PR matches what the SC have asked us to do, we'll get this approved by 360, merged and released as a MINOR upgrade.

**Comment by mrshll1001** [comment]
I've fixed the typos and addressed the points discussed.

I'll need support squashing the commits into a single commit, and then this needs reviewing.

**Comment by KDuerden** [comment]
> Reviewed with @mrshll1001 today. This is getting close to being ready.
> 
> The change has been approved by the SC, so once this PR matches what the SC have asked us to do, we'll get this approved by 360, merged and released as a MINOR upgrade.

Discussing progress on this with @mrshll1001 this AM. We have made changes to the approved schema, and drafted normative text, does it not need a post on the forum to get sign off?

**Comment by robredpath** [comment]
@KDuerden I've had a fun delve into the governance docs and yes, you're right - it's not enough that we're carrying out what the SC asked us to, there is a definite stage on the flowchart where the actual schema and docs that will get merged need to be reviewed by the "standard governance group". 

I think a bit of attention to our governance documentation wouldn't go amiss once we get 1.1 out!

**Comment by KDuerden** [comment]
great, thanks for checking and confirming. So as next steps, once you're all ready, I'll review in case there is anything further, then post on forum for approval, then we can merge. Is there going to be a branch that committee members will be looking at and I can point to the changes?

**Comment by robredpath** [comment]
> Is there going to be a branch that committee members will be looking at and I can point to the changes?

Yeah, we need to make a clean branch in this PR that makes the actual changes clear; @mrshll1001 do you want to put something in my calendar for us to go through this? Cleaning up git branches can be awkward, especially for very long-lived ones ("kindly and others added 19 commits on 12 Jul 2018 " O_o), so it might actually be easier to move the work over to a fresh branch and re-PR. Let's see when we talk.

**Pull request closed without merging**